src/main/java/com/github/chabanenk0/Main.java

```diff
@@ -1,11 +1,103 @@
 package com.github.chabanenk0;
 
+import com.github.chabanenk0.Exception.WrongInputDataException;
+import com.github.chabanenk0.Shapes.*;
+
+import java.io.IOException;
+
 /**
  * Main class for inheritance homework
  * Created by dchabanenko on 09.11.16.
  */
 public class Main {
-    public static void main(String[] args) {
 
+    private static Circle inputCircleFromConsole()
+    {
+        Circle circle = new Circle();
+        try {
+            circle.inputFromStream(System.in, true, System.out);
```

Comment:
passing and out streams as params - good
creating objects with default parameters - feels more like a workaround. InputFromStream method can be static and return a new initialized instance